pkg/arvo/sys/vane/clay.hoon

```diff
@@ -3245,16 +3361,21 @@
     ::  ones we found before `a`.  Then convert the takos to yakis and also get
     ::  all the data in all the yakis.
     ::
+    ::  What happens if you run an %init merge on a desk that already
+    ::  had a commit?
```

Comment:
The theoretical answer would seem to be "exactly the same thing as if you ran a `%that` merge, except the commits in your previous revision(s) have been orphaned. How does that fail in practice?

Reply:
The set of yakis produced is `+reachable-takos` (set of reachable ancestors via parentage -- not looking at aeons) of `b` (often the head of the desk) subtract `+reachable-takos` of `a` (usually the last commit we know for sure the foreign ship has).  Suppose there's an `%init` merge `c` between `a` and `b`, and `x` is the aeon before `c`.  Then, the descendants of `a` whose aeon is lte `x` will *not* be produced and sent over the wire, so the foreign ship will not get all the commits it needs.